api/api.yaml

```diff
@@ -0,0 +1,619 @@
+      responses: 
+        '302':
+          description: A redirect to the authenticationresponse path, containing scope, response_type, client_id, redirect_uri, state and nonce to be used for the request
+  /verifier/api/v1/receivecredential/{state}:
+    get:
+      tags:
+        - verifier
+        - frontend
+      parameters:
+        - $ref: '#/components/parameters/State'
+      operationId: VerifierPageReceiveCredential
+      summary: Presents a rendered page, showing the credential associated with the requested state
+      responses:
+        '200':
+          description: A rendered page, showing the credential
+          content:
+            text/html:
+              schema:
+                type: object
+  /verifier/api/v1/accessprotectedservice:
+    get:
+      tags:
+        - verifier
+        - frontend
+      operationId: VerifierPageAccessProtectedService
+      summary: Presents a rendered page, containing the response from the protected service
+      description: When called, the vc connected to the current session is used to access the 'protected-service'. The response then is rendered into an html page.
+      responses:
+        '200':
+          description: A rendered page, showing the response from the protected service
+          content:
+            text/html:
+              schema:
+                type: object
+  /verifier/api/v1/poll/{state}:
+    get:
+      tags:
+        - verifier
+        - api
+      parameters:
+        - $ref: '#/components/parameters/State'
+      operationId: VerifierAPIPoll
+      summary: Endpoint to check the current state of a session
+      responses:
+        '200':
+          description: Returns either the state of the session(pending, expired) or the credential itself
+          content:
+            text/plain:
+              schema:
+                type: string
+  /verifier/api/v1/token/{state}:
+    get:
+      tags:
+        - verifier
+        - api
+      parameters:
+        - $ref: '#/components/parameters/State'
+      operationId: VerifierAPIToken
+      summary: Return the jwt for the requested session
+      responses:
+        '200':
+          description: The access token
+          content:
+            text/plain:
+              schema: 
+                type: string
+        '403':
+          description: Unauthorized, in case the session is not authorized
+  /verifier/api/v1/startsiop:
+    get:
+      tags:
+        - verifier
+        - api 
+      parameters:
+        - $ref: '#/components/parameters/QueryState'
+      operationId: VerifierAPIStartSIOP
+      summary: Initiates the siop flow and returns the 'openid://...' connection string
+      responses:
+        '200': 
+          description: The connection string 
+          content:
+            text/plain:
+              schema:
```

Comment:
Is the connection string already specified somewhere? It could make sense to add the option for returnig the same info as json, which makes it easier to use and parse for a client application.